admin/templates/login.html

```diff
@@ -0,0 +1,67 @@
+{% load render_bundle from webpack_loader %}
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>OSFAdmin | Log in</title>
+    <!-- Tell the browser to be responsive to screen width -->
+    <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
+    <!-- Bootstrap 3.3.4 -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet" type="text/css" />
+    <!-- Font Awesome Icons -->
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+    <!-- Theme style -->
+    <link href="{% static 'css/AdminLTE.min.css' %}" rel="stylesheet" type="text/css" />
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+  <body class="login-page">
+    <div class="login-box">
+      <div class="login-logo">
+        <a href="../../index2.html"><b>OSF</b>Admin</a>
+      </div><!-- /.login-logo -->
+      {% if messages %}
+      {{message.count}}
+        {% for message in messages %}
+          <span class="text-success"> {{ message }}</span>
+        {% endfor %}
+      {% endif %}
+      <div class="login-box-body">
+        <p class="login-box-msg">Sign in to start your session</p>
+        <form action="" method="post">
+          {% csrf_token %}
+          {% if form.non_field_errors %} <span class="text-danger"> {{ form.non_field_errors | striptags }} </span> {% endif %}
+          <div class="form-group">
+            <label for="username">Username:</label>
+            <p>{{ form.username }}</p>
+            {% if form.username.errors %}<span class="text-danger"> {{ form.username.errors }}</span>{% endif %}
+          </div>
+          <div class="form-group">
+            <label for="username">Password:</label>
+            <p>{{ form.password }}</p>
+            {% if form.password.errors %}<span class="text-danger"> {{ form.password.errors }}</span>{% endif %}
+          </div>
+          <div class="row">
+            <div class="col-xs-4">
+              <button type="submit" class="btn btn-primary btn-block btn-flat">Sign In</button>
+            </div><!-- /.col -->
+          </div>
+        </form>
+      </div><!-- /.login-box-body -->
+    </div><!-- /.login-box -->
+
+        <!-- jQuery 2.1.4 -->
+    <script src="{% static 'js/jquery.min.js' %}"></script>
```

Comment:
If you inherit from base.html you're going to have jQuery in the page twice unless you kill this line.